trashcli/_shtab.py

```diff
@@ -0,0 +1,8 @@
+FILE = None
+DIRECTORY = DIR = None
+
+
+def add_argument_to(parser, *args, **kwargs):
+    from argparse import Action
+    Action.complete = None  # type: ignore
+    return parser
```

Comment:
At the present if I run `trash-xxx --print-completion bash` when shtab library is not available I get this error:
```
$ ./trash-list --print-completion bash
usage: trash-list [-h] [--version] [--volumes] [--trash-dirs]
                  [--trash-dir TRASH_DIRS] [--all-users]
trash-list: error: unrecognized arguments: --print-completion bash
```

Instead of showing an error can we make that it shows an hint about installing shtab?

Reply:
OK. `Please install shtab firstly!`